src/components/user/InputLoginUser.jsx

```diff
@@ -0,0 +1,36 @@
+import { Image } from "@chakra-ui/image";
+import { Input } from "@chakra-ui/input";
+import { Flex } from "@chakra-ui/layout";
+import lock from "../../assets/lock.svg";
+import user from "../../assets/user.svg";
+import React from "react";
+
+function InputLoginUser({ inputType }) {
+  return (
+    <Flex
+      border="1px solid #B6C2E2"
+      padding={5}
+      borderRadius={6}
+      alignItems="center"
+      w="100%"
+      maxW={490}
+    >
+      <Image
+        src={inputType === "password" ? lock : user}
```

Comment:
I would pass the src as a prop

```suggestion
        src={iconSrc}
```